fe/fe-core/src/main/java/org/apache/doris/planner/external/hudi/HudiCachedPartitionProcessor.java

```diff
@@ -74,6 +73,31 @@ public void cleanTablePartitions(String dbName, String tblName) {
                 .forEach(partitionCache::invalidate);
     }
 
+    public TablePartitionValues getSnapshotPartitionValues(HMSExternalTable table,
+            HoodieTableMetaClient tableMetaClient, String timestamp) {
+        assert (catalogId == table.getCatalog().getId());
```

Comment:
Prefer to use `Preconditions.checkState()`

Reply:
Done